src/js/hooks/productSupplier/form/usePreferenceTypeVariationsColumns.jsx

```diff
@@ -31,7 +31,6 @@ const usePreferenceTypeVariationsColumns = ({
   const translate = useTranslate();
 
   useEffect(() => {
-    dispatch(fetchPreferenceTypes());
     dispatch(fetchBuyers());
   }, []);
```

Comment:
This is not a change request, but rather a loud thinking. Maybe we should create a separate hook, for fetching data, we can pass functions, as an argument, and within the hook, we will check if everything is fetched, and based on that show / hide the spinner.

Reply:
I'd suggest thinking asap about the react-query that would handle such things for us (loading (spinner), caching, etc.) 
cc @drodzewicz